### DISCUSSION TITLE
中英文混合的图片无法识别英文之间的空格

### DISCUSSION THREAD

**Discussion by lucasjinreal** [PaddlePaddle/PaddleOCR#13200] [Q&A]
设置了use_space_char依旧无效，v4模型。

**Comment by SWHL** [comment]
这个我之前也发现了。应该是中英文混合训练数据较少导致模型识别效果不太行

**Comment by GreatV** [comment]
在自己的场景下，再训练一下。

**Reply by lucasjinreal** [reply]
离谱啊，我只是一个使用者，不懂ocr，一个这么牛逼的库，竟然不支持中英文混合，吐血。。

**Reply by GreatV** [reply]
你的图是啥样的，能共享几张吗

**Reply by lucasjinreal** [reply]
随便一个截图，包含中英文，用默认的模型都识别不了

**Reply by lucasjinreal** [reply]
有什么办法，通过调整参数支持i一下空格吗？我主要是一张图片里面都有中文和英文，英文不多，但是不能没有空格，其次批量处理的时候，有可能是中文图片也可能是英文图片。

主要是产出数据给多模态训练，paddleocr为啥不支持中英文混合，比较原始啊

**Comment by lucasjinreal** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/21303438/b1b33e85-9dc0-4367-943a-be730ab21a19)
zhe菊花如何理解？@paddle
